.htmlvalidate.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["html-validate:a17y"]
```

Comment:
The folks at HTML-validate recommend also including the `html-validate:standard` preset (rationale [here](https://html-validate.org/rules/presets.html#html-validate-a17y)).  That may be overkill, though...

Reply:
We wanted to reduce the noise (188 errors vs. 17 errors on this one directory) for our initial experimentation.